templates/options.conf.epp

```diff
@@ -0,0 +1,51 @@
+directory "<%= $dns::vardir %>";
+<% if $dns::forwarders and $dns::forwarders != [] { -%>
+forwarders { <%= join($dns::forwarders, '; ') %>; };
+<% } -%>
+<% if $dns::forward != undef and $dns::forward != '' { -%>
+forward <%= $dns::forward %>;
+<% } -%>
+
+<% if $dns::rpz_zones { -%>
```

Comment:
```suggestion
<% unless empty($dns::rpz_zones) { -%>
```

Reply:
happy to make this change, is there a reason unless empty is a better approach ?